src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/events/ActionPlanActionPlanAppointmentEventPublisherTest.kt

```diff
@@ -0,0 +1,84 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.events
+
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.argumentCaptor
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.whenever
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.context.ApplicationEventPublisher
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.component.LocationMapper
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Attended.LATE
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.util.ActionPlanSessionFactory
+import java.net.URI
+import java.time.OffsetDateTime
+
+class ActionPlanActionPlanAppointmentEventPublisherTest {
```

Comment:
Class name should be `ActionPlanAppointmentEventPublisherTest`

Reply:
Nice catch